pyaedt/misc/misc.py

```diff
@@ -16,3 +16,50 @@ def list_installed_ansysem():
         )
 
     return version_list
+
+
+def installed_versions():
+    """Installed AEDT versions.
```

Comment:
```suggestion
    """Get installed AEDT versions.
```
Method descriptions begin with a verb.